### ISSUE TITLE
Video search in uploaded videos

### ISSUE THREAD

**Issue by luke-nehemedia** [dukt/videos#41]
Hello

Would it be possible to enable searching for the uploaded Vimeo videos? Since the search bar is available when clicking on "Library / Uploads" my client was expecting that he would be able to search his uploaded videos. However, the search seems to work only globally on Vimeo. 

It would be great if this feature could be added to the plugin.

**Comment by agrigg** [comment]
I ran into the same scenario, having the option to limit the search to the connect Vimeo account would be great. I agree that makes sense as the default.

**Comment by luke-nehemedia** [comment]
Any update on this?

**Comment by sidm1983** [comment]
I agree with this as well. I was definitely not expecting it to do a global vimeo search. The search bar should just search within the folder it is currently in.

**Comment by luke-nehemedia** [comment]
Is there any update on that? For a paid plugin I would really expect a little bit more engagement with the users.

**Comment by espensgr** [comment]
We had the same experience with our client, it is confusing.
I see this is an old issue, will this ever be added to the plugin?
